pyomo/contrib/parmest/parmest.py

```diff
@@ -607,21 +649,56 @@ def _Q_at_theta(self, thetavals):
                     # DLW: Aug2018: not distinguishing "middlish" conditions
                     if WorstStatus != pyo.TerminationCondition.infeasible:
                         WorstStatus = results.solver.termination_condition
-                
+                    if initialize_parmest_model:
+                        if self.diagnostic_mode:
+                            print("Scenario {:d} infeasible with initialized parameter values".format(snum))
+                else:
+                    if initialize_parmest_model:
+                        if self.diagnostic_mode:
+                            print("Scenario {:d} initialization successful with initial parameter values".format(snum))
+                if initialize_parmest_model:
+                    # unfix parameters after initialization
+                    for theta in theta_init_vals:
+                        theta.unfix()
+                    scen_dict[sname] = instance
             objobject = getattr(instance, self._second_stage_cost_exp)
             objval = pyo.value(objobject)
             totobj += objval
             
         retval = totobj / len(senario_numbers) # -1??
+        if initialize_parmest_model:
+            # create extensive form of the model using scenario dictionary
+            if len(scen_dict) > 0:
+                for scen in scen_dict.values():
+                    scen._mpisppy_probability = 1 / len(scen_dict)
+            
+            if use_mpisppy:
+                ef = sputils._create_EF_from_scen_dict(scen_dict,
```

Comment:
This line is not covered with testing, which makes me suspect the tests are not considering `initialize_parmest_mode=True` and `mpi-sppy`. Does the testing harness have `mpi-sppy` installed?

Reply:
I don't think our testing harness has mpi-sppy installed

Reply:
I just approved the PR then. No `mpi-sppy` in the testing harness explains the code coverage.